.github/workflows/dispatch-update-to-build-ci.yml

```diff
@@ -14,20 +14,12 @@ jobs:
     runs-on: ubuntu-latest
     if: github.ref_type == 'tag'
     steps:
-      - uses: actions/checkout@v3
-      
-      - name: Get names of all package directories
-        id: directories
-        run: |
-          # list all the packages, turn it into a space-separated list!
-          echo "modified=$(ls packages | tr '\n' ' ')" >> $GITHUB_OUTPUT
-
       - name: Send event and tag to build-ci repo
         run: |
           curl -XPOST -u "${{ secrets.RELEASE_BUILD_CI_PAT_USERNAME }}:${{ secrets.RELEASE_BUILD_CI_PAT }}" \
             -H "Accept: application/vnd.github.everest-preview+json" \
-            -H "Content-Type: application/json" https://api.github.com/repos/ACCESS-NRI/build-ci/actions/workflows/notified-of-spack-packages-update.yml/dispatches \
-            --data '{"ref": "main", "inputs":{"spack-packages-version": "${{ github.ref_name }}", "packages": "${{ steps.directories.outputs.modified }}"}}'
+            -H "Content-Type: application/json" https://api.github.com/repos/ACCESS-NRI/build-ci/actions/workflows/build-and-push-image-build.yml/dispatches \
+            --data '{"ref": "main", "inputs":{"spack-packages-build-required": "true", "spack-packages-version": "${{ github.ref_name }}", "model": "all"}}'
```

Comment:
I wonder if we need to specify `"model": "all"` here. I'm normally a fan of being specific about things and not relying on defaults, but it seems to be in this case it might be sufficient to let the CI decide what it wants to build by default.

I can imagine a time when we no longer build a model routinely, and we might not want `all` here. Of course `all` could be redefined in the CI repo.

So I'm not super-worried about it either way, just wanted to make sure the options had been canvassed.

Reply:
Thats fair enough. I'm going to be thinking of a way to determine which is access-om2 vs access-om3 based on the directories pushed. Might ask @harshula about ways to do this programmatically based on the things in the `package.py`s so we don't have to update a file or something every time a new `package.py` is added.

Reply:
Resolved offline with @aidanheerdegen and @harshula - Will rebuild every time, as we need to be able to clone a tag and have it all there. I'll raise a ticket for it as an investigation, however.